conceptnet5/builders/raw_glove.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
I wouldn't say that a shell script belongs here, especially not one that has to be run out of the current directory. Maybe this belongs as a Ninja build step?